test/hotspot/jtreg/serviceability/jvmti/HeapMonitor/MyPackage/HeapMonitor.java

```diff
@@ -252,10 +252,9 @@ private static boolean getCheckLines() {
 
       VMOption enableJVMCI = bean.getVMOption("EnableJVMCI");
       VMOption useJVMCICompiler = bean.getVMOption("UseJVMCICompiler");
-      String compiler = System.getProperty("jvmci.Compiler");
 
       checkLines = !(enableJVMCI.getValue().equals("true")
-          && useJVMCICompiler.getValue().equals("true") && compiler.equals("graal"));
+          && useJVMCICompiler.getValue().equals("true"));
```

Comment:
Is it possible to use `jdk.test.whitebox.code.Compiler.isGraalEnabled()` here instead?

Reply:
To use whitebox I will have to update the config of every test here, which I think is too verbose:
```
diff --git a/test/hotspot/jtreg/serviceability/jvmti/HeapMonitor/MyPackage/HeapMonitorStatIntervalTest.java b/test/hotspot/jtreg/serviceability/jvmti/HeapMonitor/MyPackage/HeapMonitorStatIntervalTest.java
index e08454a4857..f086f744965 100644
--- a/test/hotspot/jtreg/serviceability/jvmti/HeapMonitor/MyPackage/HeapMonitorStatIntervalTest.java
+++ b/test/hotspot/jtreg/serviceability/jvmti/HeapMonitor/MyPackage/HeapMonitorStatIntervalTest.java
@@ -27,11 +27,15 @@ package MyPackage;
 /**
  * @test
  * @summary Verifies the JVMTI Heap Monitor sampling interval average.
- * @build Frame HeapMonitor
+ * @library /test/lib
+ * @build Frame HeapMonitor jdk.test.whitebox.WhiteBox
  * @compile HeapMonitorStatIntervalTest.java
  * @requires vm.jvmti
  * @requires vm.compMode != "Xcomp"
- * @run main/othervm/native -agentlib:HeapMonitorTest MyPackage.HeapMonitorStatIntervalTest
+ * @run driver jdk.test.lib.helpers.ClassFileInstaller jdk.test.whitebox.WhiteBox
+ * @run main/othervm/native -agentlib:HeapMonitorTest -Xbootclasspath/a:.
+ *                   -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI
+ *                   MyPackage.HeapMonitorStatIntervalTest
  */
```

Reply:
Ok.